js/widgets/pagecontainer.js

```diff
@@ -20,7 +20,24 @@ define( [
 
 	$.widget( "mobile.pagecontainer", {
 		options: {
-			theme: "a"
+			theme: "a",
+			changeOptions: {
+				transition: undefined,
+				reverse: false,
+				changeHash: true,
+				fromHashChange: false,
+
+				// By default we rely on the role defined by the @data-role attribute.
+				role: undefined,
+				duplicateCachedPage: undefined,
+				pageContainer: undefined,
```

Comment:
This makes no sense you can call change on one page container to change another.